automatic/logfusion/update.ps1

```diff
@@ -0,0 +1,37 @@
+﻿Import-Module AU
+
+function global:au_BeforeUpdate {
+    $Latest.ChecksumType32 = 'sha256'
+
+    Get-RemoteFiles -Purge
```

Comment:
Don't use `Get-RemoteFiles` when there isn't a need to do that.
Use either AU's automatic checksums, or the function `Get-RemoteChecksum`